docs/velox-backend-support-progress.md

```diff
@@ -104,8 +104,7 @@ Gluten supports 164 functions. (Draw to right to see all data types)
 | >                             | gt                     | greaterthan           | S      |                        | S       | S    | S     | S   | S    | S     | S      |      |           | S      |         |      |        |          |       |      |        |      |
 | >=                            | gte                    | greaterthanorequal    | S      |                        | S       | S    | S     | S   | S    | S     | S      |      |           | S      |         |      |        |          |       |      |        |      |
 | ^                             | bitwise_xor            |                       | S      |                        |         |      | S     | S   | S    |       |        |      |           |        |         |      |        |          |       |      |        |      |
-| \                             |                        | bitwise_or            | bitwise_or            | S      |                        |         |      | S     | S   | S    |       |        |      |           |        |         |      |        |          |       |      |        |      |
```

Comment:
Yes, the form was incorrectly changed at https://github.com/oap-project/gluten/commit/adb1bf05038cb60c10756936280bc624ee3d3c0e.
It looks we have to keep `\` as escape char to present `|` symbol for this function. Could you refine this patch? Thanks!